BitBar/version.12h.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
```

Comment:
This file is not executable. Please `chmod +x` and commit the permissions (`git add BitBar/version.12h.rb`)